R/dist.R

```diff
@@ -205,8 +205,7 @@ dist_fit <- function(values = NULL, samples = NULL, cores = 1,
 
   if (samples < 1000) {
     samples <- 1000
-  }
-
+    }
```

Comment:
this seems like it isn't a required or desired change?

Reply:
I reverted this change in 7b63d9bf79b1c30ae15df7780e2a013b33223927. Are you reviewing sequentially by commit?